config/Snyk_PoC_results.csv

```diff
@@ -716,34 +737,8 @@ CVE-2023-45805;SNYK-PYTHON-UNEARTH-6026479;https://security.snyk.io/vuln/SNYK-PY
 CVE-2023-45805;SNYK-PYTHON-PDM-6026478;https://security.snyk.io/vuln/SNYK-PYTHON-PDM-6026478;yes;no;no;no;
 CVE-2023-45322;SNYK-UNMANAGED-LIBXML2-5952341;https://security.snyk.io/vuln/SNYK-UNMANAGED-LIBXML2-5952341;yes;no;no;no;
 CVE-2023-45292;SNYK-GOLANG-GITHUBCOMMOJOCNBASE64CAPTCHA-6120445;https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMMOJOCNBASE64CAPTCHA-6120445;yes;no;no;no;
-CVE-2023-45284;SNYK-SLES155-GO121OPENSSLRACE-6067689;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSLRACE-6067689;yes;no;no;no;
-CVE-2023-45284;SNYK-SLES155-GO121OPENSSLDOC-6067693;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSLDOC-6067693;yes;no;no;no;
-CVE-2023-45284;SNYK-SLES155-GO121OPENSSL-6067718;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSL-6067718;yes;no;no;no;
-CVE-2023-45284;SNYK-SLES154-GO121OPENSSLRACE-6067634;https://security.snyk.io/vuln/SNYK-SLES154-GO121OPENSSLRACE-6067634;yes;no;no;no;
-CVE-2023-45284;SNYK-SLES154-GO121OPENSSLDOC-6067690;https://security.snyk.io/vuln/SNYK-SLES154-GO121OPENSSLDOC-6067690;yes;no;no;no;
-CVE-2023-45283;SNYK-SLES155-GO121OPENSSLRACE-6067689;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSLRACE-6067689;yes;no;no;no;
-CVE-2023-45283;SNYK-SLES155-GO121OPENSSLDOC-6067693;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSLDOC-6067693;yes;no;no;no;
-CVE-2023-45283;SNYK-SLES155-GO121OPENSSL-6067718;https://security.snyk.io/vuln/SNYK-SLES155-GO121OPENSSL-6067718;yes;no;no;no;
-CVE-2023-45283;SNYK-SLES154-GO121OPENSSLRACE-6067634;https://security.snyk.io/vuln/SNYK-SLES154-GO121OPENSSLRACE-6067634;yes;no;no;no;
-CVE-2023-45283;SNYK-SLES154-GO121OPENSSLDOC-6067690;https://security.snyk.io/vuln/SNYK-SLES154-GO121OPENSSLDOC-6067690;yes;no;no;no;
 CVE-2023-45277;SNYK-JAVA-ORGYAMCS-6026729;https://security.snyk.io/vuln/SNYK-JAVA-ORGYAMCS-6026729;yes;no;no;no;
 CVE-2023-45197;SNYK-PHP-VRANAADMINER-7361755;https://security.snyk.io/vuln/SNYK-PHP-VRANAADMINER-7361755;no;no;yes;no;
-CVE-2023-45143;SNYK-SLES155-NPM18-6016518;https://security.snyk.io/vuln/SNYK-SLES155-NPM18-6016518;yes;no;no;no;
```

Comment:
wondering why we are removing all these entries.
@HoxhaEndri could you check this please

Reply:
https://security.snyk.io/vuln/SNYK-SLES155-NPM18-6016518
https://security.snyk.io/vuln/SNYK-SLES155-PROTOBUFDEVEL-6261762